### DISCUSSION TITLE
Github Contributions disappeared

### DISCUSSION THREAD

**Discussion by iamgauravpant** [community/community#35907] [New to GitHub]
Hi
I see that my contributions have disappeared from Github ( around 160+ contributions in total ) that were made to the company's repository from my Github account . I use Windows OS. I used to work in pc provided by them as well as from my own laptop ( even commits made from my laptop is not showing there ) . I've left the company and they've removed my access from the repository. https://www.youtube.com/watch?v=WgZIv5HI44o ->  this is the video from where I learnt how to setup SSH keys for Github.

While searching out I found this in Github docs : "If you are part of an organization that uses SAML single sign-on (SSO), you won’t be able to see contribution activity from the organization on your profile if you do not have an active SSO session." Github docs link : https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile Any help would be appreciated ( I don't know much about Github ).

**Comment by queenofcorgis** [comment]
Hi there! 

Is your work email still attached to your account?

**Reply by AF250329** [reply]
Did you understand the question ? What help this person want ? He just bring the document that says that all his contribution will be removed and then asking for help.. Help in what ? Return the contribution ? I didn't really understand what actually he want

**Reply by AF250329** [reply]
As I understand from question - he was working in his **company private** repo  - now he is not working there. I guess for the best reasons his company won't provide him an access to this repo any more. So what does it mean *help to get back to his full repo* ?

**Comment by radar-blip** [comment]
I'm guessing since it was the companies repository, which you dont have access to anymore, that's probably why contributions aren't showing up on your side (on your profile). Those contributions will probably show up on the companies side.

**Comment by Dare-marvel** [comment]
I made a change to my github readme on 16th april which was previously showing up but few days later it disappeared all of a sudden .... I don't know how

![image](https://github.com/community/community/assets/104304028/06aa8f56-e3c2-4bc5-b165-d3b40a45810e)

This was my previous contribution graph 

This is the new one with the missing contribution 

![image](https://github.com/community/community/assets/104304028/03142524-6c8c-445a-912d-e6babdf0a723)

**Reply by theyonatan** [reply]
having this as well.
did you solve it?

it seems like even new contributions I make don't get counted.
